compose/install.md

```diff
@@ -95,185 +92,120 @@ on Microsoft Windows Server and want to install Docker Compose.
     ```
 
     Then run the following command to download the current stable release of
-    Compose (v{{site.compose_version}}):
+    Compose ({{site.compose_version}}):
 
     ```powershell
     Invoke-WebRequest "https://github.com/docker/compose/releases/download/{{site.compose_version}}/docker-compose-Windows-x86_64.exe" -UseBasicParsing -OutFile $Env:ProgramFiles\Docker\docker-compose.exe
     ```
 
-**Note**: On Windows Server 2019, you can add the Compose executable to `$Env:ProgramFiles\Docker`. Because this directory is  registered in the system `PATH`, you can run the `docker-compose --version` command on the subsequent step with no additional configuration.
+    >**Note**:  
+    > On Windows Server 2019, you can add the Compose executable to `$Env:ProgramFiles\Docker`. Because this directory is  registered in the system `PATH`, you can run the `docker-compose --version` command on the subsequent step with no additional configuration.
 
-    > To install a different version of Compose, substitute `{{site.compose_version}}`
+    > To install a different version of Compose, substitute `{{site.compose_version}}` 
     > with the version of Compose you want to use.
 
 3.  Test the installation.
 
     ```powershell
-    docker-compose --version
-
-    docker-compose version {{site.compose_version}}, build 01110ad01
+    $ docker compose --version
```

Comment:
Isn't it `docker compose version` instead ?

Reply:
nice catch 👍